agent/pkg/api/service_map.go

```diff
@@ -0,0 +1,217 @@
+package api
```

Comment:
we should create a separate package for the service functionality like "agent/pkg/serviceMap" with all related files